Encounter Me/Services/ICapturePointService.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+using BrowserInterop.Geolocation;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```

Comment:
Don't you get an error or warning for repeating `using` statements?

Reply:
fixed in newest commit